CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # Unreleased
+### Changed
+- Upgrade MSAL to 4.47.2.
+- Opt-in native WAM mode and Windows SDK is removed.
+- MSALWrapper now use package reference directly.
```

Comment:
This is slightly too much detail for the change log. The main target of the change log is a consumer of AzureAuth and they want to know quickly,  is anything breaking, are there any new features, is anything deprecated. Here, I think 1 line to note the update to MSAL 4.47 and opt-into native broker mode (which is expected to have no negative impact on AzureAuth's current functionality) is good.